### ISSUE TITLE
osquery-go without watchdog.

### ISSUE THREAD

**Issue by cssmason** [osquery/osquery-go#93]
There is no watchdog be launched unless the enable_extension_watchdog=true

**Comment by directionless** [comment]
I am not sure I understand this issue. Could you explain more?

**Comment by cssmason** [comment]
There is one  configuration “enable_extensions_watchdog” in osquery.
I have enable this feature in osquery flag file. But when i launched osquery, there is no watchdog process being launched.

**Comment by directionless** [comment]
How are you running osquery? This repo is the go SDK, while it can pass that along as a config option, it does not manage an osquery install.

**Comment by cssmason** [comment]
Yes, I use osquery golang SDK to develop the extension.exe. I launched osqueryd with this extension.exe and the flags "enable_extension_watchdog=true". But I can not see any watchdog thread of this extension.exe be launched.
And I saw the osquery-python repo, there are two features for the watchdog of extension.
Ref: https://github.com/osquery/osquery-python/pull/31
But I can not see any code about this feature in osquery-go.
Could development team help to enhance this feature?
